xbmc/filesystem/FavouritesDirectory.cpp

```diff
@@ -22,219 +22,31 @@
 #include "File.h"
 #include "Directory.h"
 #include "Util.h"
-#include "interfaces/AnnouncementManager.h"
 #include "profiles/ProfilesManager.h"
-#include "FileItem.h"
-#include "utils/XBMCTinyXML.h"
-#include "utils/log.h"
-#include "utils/StringUtils.h"
 #include "utils/URIUtils.h"
-#include "settings/AdvancedSettings.h"
-#include "video/VideoInfoTag.h"
 #include "music/tags/MusicInfoTag.h"
-#include "URL.h"
 
 namespace XFILE
 {
 
+CFavouritesDirectory::CFavouritesDirectory()
+{
+}
```

Comment:
See my other comment. I would declare the ctor as <code>default</code> if the definition is actually emoty,